tsconfig.json

```diff
@@ -14,25 +14,25 @@
       "@aws-lambda/mds-native": ["aws-lambda/mds-native/index.ts"],
       "@aws-lambda/mds-policy": ["aws-lambda/mds-policy/index.ts"],
       "@aws-lambda/mds-provider": ["aws-lambda/mds-provider/index.ts"],
-      "mds-agency": ["packages/mds-agency/index.ts"],
-      "mds-api-authorizer": ["packages/mds-api-authorizer/index.ts"],
-      "mds-api-helpers": ["packages/mds-api-helpers/index.ts"],
-      "mds-api-server": ["packages/mds-api-server/index.ts"],
-      "mds-audit": ["packages/mds-audit/index.ts"],
-      "mds-cache": ["packages/mds-cache/index.ts"],
-      "mds-compliance": ["packages/mds-compliance/index.ts"],
-      "mds-daily": ["packages/mds-daily/index.ts"],
-      "mds-db": ["packages/mds-db/index.ts"],
-      "mds-logger": ["packages/mds-logger/index.ts"],
-      "mds-metrics-sheet": ["packages/mds-metrics-sheet/index.ts"],
-      "mds-native": ["packages/mds-native/index.ts"],
-      "mds-policy": ["packages/mds-policy/index.ts"],
-      "mds-provider": ["packages/mds-provider/index.ts"],
-      "mds-providers": ["packages/mds-providers/index.ts"],
-      "mds-stream": ["packages/mds-stream/index.ts"],
-      "mds-test-data": ["packages/mds-test-data/index.ts"],
-      "mds-types": ["packages/mds-types/index.ts"],
-      "mds-utils": ["packages/mds-utils/index.ts"],
+      "@mds-core/mds-agency": ["packages/mds-agency/index.ts"],
```

Comment:
Now that I think about it, with the addition of the scope, these path mappings can be reduced to:
```
"@aws-lambda/*": ["aws-lambda/*"],
"@mds-core/*": ["packages/*"],
"*": ["types/*", "*"]
```

Reply:
done